### ISSUE TITLE
Publish version 2.5

### ISSUE THREAD

**Issue by voltan** [pi-engine/pi#1304]
Hello

when do you think , we can ready for public version 2.5?

**Comment by taiwen** [comment]
Check markup branch.
Once it is merged, we can release 2.5

**Comment by voltan** [comment]
Thanks , just update translations needed

**Comment by Marc-pi** [comment]
well, if we look at Pi.2.5 milestone, 
* [ ] update translation https://github.com/pi-engine/pi/issues/1239
* [ ] finalize default theme issues https://github.com/pi-engine/pi/issues/1285 and https://github.com/pi-engine/pi/issues/1282
* [ ] markup and url clickable https://github.com/pi-engine/pi/issues/1257
* [ ] sitemap on article module  https://github.com/pi-engine/pi/issues/1235
* [ ] respond and other old browser libs update https://github.com/pi-engine/pi/issues/951
* [ ] bug on comment disable https://github.com/pi-engine/pi/issues/835

IMHO the other can be postponed and added though a maintenance intermediate release
* [ ] user module activity https://github.com/pi-engine/pi/issues/1231
* [ ] captcha https://github.com/pi-engine/pi/issues/1161
* [ ] video player https://github.com/pi-engine/pi/issues/1294
* [ ] login form consistency / templating  https://github.com/pi-engine/pi/issues/1155
* [ ] composer dependency mngr https://github.com/pi-engine/pi/issues/957
* [ ] pi website https://github.com/pi-engine/pi/issues/1211
* [ ] support litespeed https://github.com/pi-engine/pi/issues/1245

**Comment by Marc-pi** [comment]
zf2.3.4 is out

**Comment by Marc-pi** [comment]
@voltan @taiwen 
i've updated the milestone, ok for you for the dead line?

![pi2 5](https://cloud.githubusercontent.com/assets/919332/5893770/0ec2a42c-a4f1-11e4-9094-bea32d4496c2.JPG)


i've created milestone for Pi2.6 also

**Comment by voltan** [comment]
Can we have new version whit minum of options and just bug fix on next two weeks?

**Comment by voltan** [comment]
I vote to finish just this for this version  and do other for next version : 

* [ ] update translation #1239
* [ ] access denied
* [ ] search bug
* [ ] robotx.txt
* [ ] finalize default theme SEO issues #1285 and #1282
* [ ] markup and url clickable #1257
* [ ] bug on comment disable #835
* [ ] update changelog
* [ ] browser cache bug
* [ ] meta robots tag (what is this ?)
* [ ] profile sensitive info > page permission (what is this ?)

**Comment by Marc-pi** [comment]
ok for me , we please don't forget to update third part libs @taiwen ZF > session component fixes are critical for us?

**Comment by Marc-pi** [comment]
Well, the most critical issues we need to finish up are :
* browser cache bug 
* finish SEO stuff https://github.com/pi-engine/pi/issues/1341 and https://github.com/pi-engine/pi/issues/1339 and https://github.com/pi-engine/pi/issues/1253
* loading jquery& co faster https://github.com/pi-engine/pi/issues/1335
* update boostrap and perhaps ZF2 if security is a concern for our implementation (ZF2.4 is planned to be release around mid march)
* close the issues labelled bug

=> We have strong SEO and performance issues we need to tackle down. i think it is important to close those basics very very soon, our live sites need those fix.

@taiwen can you plan to work on this asap, so we close the SEO / Performance issues within a good solid 2.5 version, but soon ???
